ui/src/pages/Formation/Formation.js

```diff
@@ -175,22 +175,26 @@ export default ({ match }) => {
     },
     onSubmit: ({ uai_formation }) => {
       return new Promise(async (resolve) => {
+        const trimedUaiFormation = uai_formation?.rim();
```

Comment:
Je crois que le "t" est parti ^^
```suggestion
        const trimedUaiFormation = uai_formation?.trim();
```

Reply:
Effectivement, c'est corrigé